spec/controllers/statuses_controller_spec.rb

```diff
@@ -731,52 +731,52 @@
     end
 
     context 'when signed-in' do
-      context 'when status is public' do
+      context 'when status is public for signed in' do
```

Comment:
I'm afraid this is misleading, a status being public is a property of the status, not of whether someone is signed in.

Reply:
This one is kind of weird because this entire block of specs should probably just be deleted if they're not going to be implemented? I took a first pass at adding some of them here - https://github.com/mastodon/mastodon/pull/23969 - but need feedback.

Reply:
Oh, sorry I missed this PR, I'll try to provide guidance on it early next week!